### ISSUE TITLE
Editing existing extractor, unable to specify test message id

### ISSUE THREAD

**Issue by reighnman** [Graylog2/graylog2-web-interface#1105]
graylog2-web-interface v1.0.0-rc.2

When editing an extractor it loads a random message from the bound input to test against.  When I created the extractor I used a full_message from a specific type of log as several sources use this input.

I should be able to load a message to test regex/grok against instead of it randomly selecting a message from the input.  Or at least tie the edit extractor page to the main extractor page where you're able to load a message id when creating a new extractor.

**Comment by edmundoa** [comment]
Thank you for reporting this. The manual loading makes complete sense and it's something we should add.

**Comment by runningman84** [comment]
A button to load another random message would help too. I would like to the my patterns against a bunch of message to make sure I catch all messages...

For eaxample my pattern looks like
```
(?:dstport|dst_port|destination-port|service)(?:=|:)"?(\d{1,6})"?
```
I would need at four three messages to verify that my pattern works for all of them.

**Comment by kroepke** [comment]
fixed by merged pull request #1156  referenced above
